### PR TITLE
Login register

### PR DESCRIPTION
**Pull request by CShum28**

### CODE REVIEW

File: modules/gamelist-ui/src/lib/Button/Button.tsx [hunk 1]
```diff
@@ -14,12 +14,14 @@ const buttonVariants = cva(
...
         ghost: 'hover:bg-accent hover:text-accent-foreground',
         link: 'text-primary underline-offset-4 hover:underline',
+        loginSignUp: 'text-secondary bg-[#3db4f2] rounded-md mt-2.5 cursor-pointer font-semibold',
       },
       size: {
         default: 'h-10 px-4 py-2',
         sm: 'h-9 rounded-md px-3',
         lg: 'h-11 rounded-md px-8',
         icon: 'h-10 w-10',
+        loginSignUp: 'py-[10px] px-[30px]',
```
Comment (Yzma) [inline modules/gamelist-ui/src/lib/Button/Button.tsx]: Put something like: py-8 px-8 here, remove the hard coded px values

File: modules/gamelist-ui/src/lib/Input/Input.tsx [hunk 2]
```diff
@@ -1,19 +1,41 @@
...
 
-export type InputProps = React.InputHTMLAttributes<HTMLInputElement>;
+const inputVariants = cva(
+  'border-input bg-background ring-offset-background placeholder:text-muted-foreground focus-visible:ring-ring flex h-10 w-full rounded-md border px-3 py-2 text-sm file:border-0 file:bg-transparent file:text-sm file:font-medium focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-offset-2 disabled:cursor-not-allowed disabled:opacity-50',
+  {
+    variants: {
+      variant: {
+        default: '',
+        loginSignUp:
+          'w-[290px] px-[15px] focus:outline-none focus-visible:ring-0 focus-visible:ring-offset-0 bg-[#f2f4f7]',
```
Comment (Yzma) [inline modules/gamelist-ui/src/lib/Input/Input.tsx]: Same here, remove the hard-coded px values. Also can you remove bg-[#f2f4f7] for bg-background? Or is that specific color needed?

**Review by Yzma — APPROVED** [review]

File: modules/gamelist-ui/src/lib/Button/Button.tsx (hunk 1 above)
Comment (CShum28) [reply modules/gamelist-ui/src/lib/Button/Button.tsx]: @Yzma check this out:
[remove px values](https://github.com/tienviet10/gamelist-fe-v3/pull/16/commits/f42baa4274a47bc9ff2459f7387a844fd61f100c)

File: modules/gamelist-ui/src/lib/Input/Input.tsx (hunk 2 above)
Comment (CShum28) [reply modules/gamelist-ui/src/lib/Input/Input.tsx]: @Yzma check this out again:
https://github.com/tienviet10/gamelist-fe-v3/pull/16/commits/581dfb820e32bc4ca2843189f0540c6e22565d64

**Pull request merged**